libcodechecker/cmd/cmd_line_client.py

```diff
@@ -420,8 +420,22 @@ def print_reports(client, reports, output_format):
 
         header = ['File', 'Checker', 'Severity', 'Msg', 'Source']
         rows = []
+
+        source_lines = defaultdict(set)
+        for report in reports:
+            if not isinstance(report, Report):
```

Comment:
Why do we need this check? What type of objects can be in reports?

Reply:
We need this check to see that this data was created from a plist (new and unresolved reports):
https://github.com/Ericsson/codechecker/blob/de66e7078f285291f1fffbe7149a921eb9b86c4d/libcodechecker/report.py#L149
or this data comes from the database (in case of resolved reports):
https://github.com/Ericsson/codechecker/blob/de66e7078f285291f1fffbe7149a921eb9b86c4d/libcodechecker/server/database/run_db_model.py#L181